habit-backend/habit/settings.py

```diff
@@ -31,6 +32,7 @@
 # Application definition
 
 INSTALLED_APPS = [
+    'users',
```

Comment:
Will we be making Apps for all models?
How is it going to work?

Reply:
App is not built for a specific model. Its basically made to distinguish between modules.
In our case we will be using users app for login/ signup/password change functionality